src/app/pages/wallets/my-wallets/my-wallet/pages-wallets-my-wallets-my-wallet.component.ts

```diff
@@ -8,4 +8,10 @@ import { MyWallet } from '../pages-wallets-my-wallet.model';
 })
 export class PagesWalletsMyWalletsMyWalletComponent {
   @Input() public wallet!: MyWallet;
-}
+
+  @Output('openWalletModalCallback') openWalletModalCallback = new EventEmitter<number>();
+
+  public openEditWalletModal(): void {
+    this.openWalletModalCallback.emit(this.wallet.id);
```

Comment:
You don't need to emit `id` here. You know which wallet was clicked already in parent component :)